docs-chef-io/content/server/config_rb_server_optional_settings.md

```diff
@@ -1461,6 +1461,12 @@ This configuration file has the following settings for `oc-id`:
 
     **New in Chef Infra Server 15.9.20**
 
+`oc_id['onetrust_id']`
+
+: The Onetrust ID for Chef Infra. You must also set `oc_id['enable_onetrust']` to `true`.
```

Comment:
```suggestion
: The OneTrust data domain script ID for Chef Infra. You must also set `oc_id['enable_onetrust']` to `true`.
```

Reply:
Done

Reply:
chef-server current version is 15.9.20 we have 3 more probable PR's to merge, we can keep it 15.9.22 for now and we will change it in release.

Reply:
Sure @sreepuramsudheer I will add 15.9.22. Thanks!